src/usr/local/www/firewall_rules_edit.php

```diff
@@ -521,6 +525,10 @@ function is_aoadv_used($rule_config) {
 		$input_errors[] = gettext("Invert match cannot be selected with 'any'.");
 	}
 
+	if ((isset($_POST['nottagged']) && empty($_POST['tagged']))) {
```

Comment:
There is an extra set of parenthesis on the outside this test

Reply:
fixed